next.config.js

```diff
@@ -21,6 +21,42 @@ module.exports = {
       }
     ]
   },
+  async rewrites() {
```

Comment:
What's the purpose of rewriting the URLs?

Reply:
The goal here is to link the `FAQ`, `Terms` and `Rewards` pages to the same path under the pages folder as they are the same under the folders validators and builders program but the text content will be changed via the router path. So in order to not recreate them under the two folders, i linked them with rewriting the URLs.